config.go

```diff
@@ -60,6 +60,7 @@ type TGFConfig struct {
 	RunBefore               string            `yaml:"run-before,omitempty" json:"run-before,omitempty" hcl:"run-before,omitempty"`
 	RunAfter                string            `yaml:"run-after,omitempty" json:"run-after,omitempty" hcl:"run-after,omitempty"`
 	Aliases                 map[string]string `yaml:"alias,omitempty" json:"alias,omitempty" hcl:"alias,omitempty"`
+	LatestTgfVersion        string            `yaml:"latest-tgf-version,omitempty" json:"latest-tgf-version,omitempty" hcl:"latest-tgf-version,omitempty"`
```

Comment:
It is not the latest version necessarily. It is the version to which we want to update. It should be called `UpdateVersion`.
As we talked about, there should also be an option to disable auto update and an option to set the auto-update delay

Reply:
Oh okay so:
1- Should it be opt-in (at the config file level) ? Example: `"auto-update": "on"`
2- Should users specify the time delay (like docker-refresh?) ? Example: `"auto-update-delay": "2d"`

Reply:
1. It should be opt-out at the config-level
2. yes